packages/server/graphql/public/mutations/startRecurrence.ts

```diff
@@ -36,7 +37,27 @@ const startRecurrence: MutationResolvers['startRecurrence'] = async (
   }
 
   if (meeting.meetingSeriesId) {
-    return standardError(new Error('Meeting already has meeting series'), {userId: viewerId})
+    const meetingSeries = await dataLoader.get('meetingSeries').loadNonNull(meeting.meetingSeriesId)
+    if (!meetingSeries.cancelledAt) {
+      return standardError(new Error('Meeting is already recurring'), {userId: viewerId})
+    }
+
+    // meeting has stopped recurrence associated with it, reenable it
+    await updateMeetingSeries({cancelledAt: null}, meeting.meetingSeriesId)
```

Comment:
-1 We should update the recurrence rule to make sure the next meeting starts at a reasonable time. E.g. if this meeting originally started a few days ago, then the recurrence rule would likely prescribe a meeting in the past 24 hours, so the next `processRecurrence` run would immediately start a new meeting, which would lead to two meetings being open for 24 hours.

I'd suggest the following to handle this case (we'll probably refine these semantics over time):
* If `cancelledAt` is in the past hour or so, or the next start time defined by in the recurrence rule is at least 9 hours (arbitrarily chosen) from now, don't change the recurrence rule
  * For bonus points, also update the `scheduledEndTime` according to duration + startTime
* If `cancelledAt` is more than an hour ago, and the next start time is within the next 9 hours, change the recurrence rule's start date to be 24 hours after that next start time.

This would be in-line with the current `startRecurrence` start date semantics, which choose the following day (UTC) to have the first meeting started.

Longer term, we'll probably build more of a UI for figuring out when a user wants their next meeting to start

Reply:
@jmtaber129 thank you for catching this! I think I've covered all the cases, do you mind taking a look? thank you!